tools/SetupFlow/DevHome.SetupFlow/Views/Environments/SelectEnvironmentProviderView.xaml

```diff
@@ -23,11 +24,11 @@
                 <ItemContainer
                     AutomationProperties.Name="{x:Bind DisplayName, Mode=OneWay}"
                     IsSelected="{x:Bind IsSelected}">
-                    <toolKit:SettingsCard
```

Comment:
Is toolkit used anywhere else in this file? Can we remove it from the top?

Reply:
sure will update